src/ResourceManager/Websites/Commands.Websites/Models.WebApp/WebAppBaseClient.cs

```diff
@@ -18,35 +18,25 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using System.Management.Automation;
-using Microsoft.Azure.Management.WebSites.Models;
-using Microsoft.WindowsAzure;
-using Microsoft.WindowsAzure.Commands.Utilities.CloudService;
-using Microsoft.Azure.Commands.Websites;
-using Microsoft.Azure.Management.WebSites;
-using System.Net.Http;
-using System.Threading;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
-using System.Net;
-using Microsoft.Azure;
 using Microsoft.WindowsAzure.Commands.Utilities.Common;
 using Microsoft.Azure.Commands.Websites.Utilities;
 
-
-namespace Microsoft.Azure.Commands.Websites.Cmdlets.WebHostingPlan
+namespace Microsoft.Azure.Commands.Websites.Models.WebApp
 {
-    /// <summary>
-    /// this commandlet will let you delete an Azure Web Hosting Plan using ARM APIs
-    /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureWebHostingPlan"), OutputType(typeof(WebHostingPlanGetResponse))]
-    public class GetWebHostingPlanCmdlet : WebHostingPlanBaseCmdlet
+    public abstract class WebAppBaseClient : AzurePSCmdlet
```

Comment:
it's a comment I had before, the name client is incorrect for this class so I think it should be renamed